src/server/pkg/storage/chunk/storage.go

```diff
@@ -11,13 +12,15 @@ const (
 	prefix = "chunks"
 )
 
-// Annotation is used to associate information with a set of bytes
+// Annotation is used to associate information with data
 // written into the chunk storage layer.
 type Annotation struct {
-	Offset      int64
 	RefDataRefs []*DataRef
 	NextDataRef *DataRef
 	Meta        interface{}
```

Comment:
Can we call this `Meta` field `Data`? We had a similar conversation on another field, but `Meta` is to obscure / abstract of a name in my mind.

Reply:
Yes.